### DISCUSSION TITLE
(ignore/close pls.) Actions stopped running. They were running yesterday. No yml changes. Now they have stopped.

### DISCUSSION THREAD

**Discussion by mcapodici** [community/community#49049] [Actions] [answered]
# Update: Sorry, ignore, this is already reported on https://www.githubstatus.com/

### Select Topic Area

Bug

### Body

See https://github.com/mcapodici/firestarter/commits/main

Commits up to and including b554d1eb92ea78c1d8278605620162135bf77ba5 have actions run on them, after this they are not. No yml changes were made to cause this.

Is it simply I have run out of free credit or something? If it is, please make it more clear *why* the actions are not running.

**Comment by RTS-Galaxies** [comment, marked as answer]
My actions is not working as well

**Reply by mcapodici** [reply]
You saying that made me check https://www.githubstatus.com/, and indeed "[Incident with Actions and Pages](https://www.githubstatus.com/incidents/qfby3shb6r3w)". I might as well close this issue.
